### PR TITLE
Update wechat desktop shortcut

### PR DESCRIPTION
**Pull request by mobilegmYT**
As it turns out, the shortcuts went to a .ink file which didn't exist. This will sed the shortcuts to the exe directly.

Also makes a warning about mono and stuff which I don't think is bypassable.

### CODE REVIEW

**Comment by Botspot** [comment]
Two different kinds of syntax errors. :(

**Comment by mobilegmYT** [comment]
> Two different kinds of syntax errors. :(

What errors?

**Comment by Botspot** [comment]
> > Two different kinds of syntax errors. :(
> 
> What errors? @Botspot

Line 17: look at your quotation marks.  
Line 42-43: notice that you're trying to use environment variables inside single quotes, which means they won't be expanded by the shell.

**Comment by mobilegmYT** [comment]
> Line 42-43: notice that you're trying to use environment variables inside single quotes, which means they won't be expanded by the shell.

So do I like change it to ~/ or something?

File: apps/Wechat/install-32 [hunk 1]
```diff
@@ -14,6 +14,9 @@ if ! command -v wine || [ "$(cat "${DIRECTORY}/data/status/Wine (x86)" 2>/dev/nu
 fi
 ulimit -c unlimited
 
+echo 'Note that during this installation, a error may pop up saying "Geco" or "Mono" isn't installed. Don't worry, that error is normal. Just click the "Install" button!'
```
Comment (Itai-Nelken) [inline apps/Wechat/install-32:17]:
```suggestion
echo "Note that during this installation, a error may pop up saying \"Geco\" or \"Mono\" isn't installed. Don't worry, that error is normal. Just click the \"Install\" button!"
```

File: apps/Wechat/install-32 [hunk 2]
```diff
@@ -36,5 +39,8 @@ wget -O ~/WeChatSetup.exe https://dldir1.qq.com/weixin/Windows/WeChatSetup.exe
 WINEPREFIX=~/.config/wechat-prefix wine ~/WeChatSetup.exe
 rm ~/WeChatSetup.exe
 
+sudo sed -i 's|/home/pi/.config/wechat-prefix/dosdevices/c:/users/Public/Desktop/WeChat.lnk|$HOME/.config/wechat-prefix/dosdevices/c:/Program Files/Tencent/WeChat/WeChat.exe|g' $HOME/Desktop/WeChat.desktop
```
Comment (Itai-Nelken) [inline apps/Wechat/install-32:42]:
```suggestion
sudo sed -i "s|$HOME/.config/wechat-prefix/dosdevices/c:/users/Public/Desktop/WeChat.lnk|$HOME/.config/wechat-prefix/dosdevices/c:/Program Files/Tencent/WeChat/WeChat.exe|g" $HOME/Desktop/WeChat.desktop
```
this should work, but isn't tested.

File: apps/Wechat/install-32 [hunk 3]
```diff
@@ -36,5 +39,8 @@ wget -O ~/WeChatSetup.exe https://dldir1.qq.com/weixin/Windows/WeChatSetup.exe
...
+sudo sed -i 's|/home/pi/.config/wechat-prefix/dosdevices/c:/users/Public/Desktop/WeChat.lnk|$HOME/.config/wechat-prefix/dosdevices/c:/Program Files/Tencent/WeChat/WeChat.exe|g' $HOME/.local/share/applications/wine/Programs/WeChat/WeChat.desktop
```
Comment (Itai-Nelken) [inline apps/Wechat/install-32:43]:
```suggestion
sudo sed -i "s|$HOME/.config/wechat-prefix/dosdevices/c:/users/Public/Desktop/WeChat.lnk|$HOME/.config/wechat-prefix/dosdevices/c:/Program Files/Tencent/WeChat/WeChat.exe|g" $HOME/.local/share/applications/wine/Programs/WeChat/WeChat.desktop
```
again, should work but not tested.

**Comment by Botspot** [comment]
> > Line 42-43: notice that you're trying to use environment variables inside single quotes, which means they won't be expanded by the shell.
> 
> So do I like change it to ~/ or something?

No, that wouldn't work either.

@mobilegmYT I get the impression that you're unable to _test_ your code. Is your Pi broken or something? Anything I can do for you?

**Comment by mobilegmYT** [comment]
@Botspot Could you test n' merge?

**Comment by mobilegmYT** [comment]
@Botspot

**Comment by mobilegmYT** [comment]
@Botspot All changes here have been tested.

**Comment by Botspot** [comment]
> @Botspot All changes here have been tested.

Are you sure? See line 55.

**Comment by mobilegmYT** [comment]
@Botspot now...

**Comment by mobilegmYT** [comment]
image
Tested to work now @Botspot.

**Pull request merged**